etl/steps/data/meadow/who/2023-04-03/flu_elderly.py

```diff
@@ -0,0 +1,49 @@
+"""Load a snapshot and create a meadow dataset."""
+
+import pandas as pd
+from owid.catalog import Table
+from structlog import get_logger
+
+from etl.helpers import PathFinder, create_dataset
+from etl.snapshot import Snapshot
+
+# Initialize logger.
+log = get_logger()
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    log.info("flu_elderly.start")
+
+    #
+    # Load inputs.
+    #
+    # Retrieve snapshot.
+    snap: Snapshot = paths.load_dependency("flu_elderly.xlsx")
+
+    # Load data from snapshot.
+    df = pd.read_excel(snap.path)
+    df = df[:-1]
```

Comment:
Maybe you can comment on these steps to come back and not forget what you were doing